README.rst

```diff
@@ -61,28 +61,44 @@ Examples_ that demonstrate how to use it.
 .. _Examples: https://skypy.readthedocs.io/en/stable/examples/index.html
 
 
-Get in Touch
+Contributing
 ------------
 
-You are welcome to talk about the SkyPy package and code using our
-`Discussions Page`_. For any other questions about the project in general,
-please get in touch with the `SkyPy Co-ordinators`_.
+We love contributions! 
+SkyPy is open source,
+built on open source, and we'd love to have you hang out in our community.
 
- .. _Discussions Page: https://github.com/skypyproject/skypy/discussions
- .. _SkyPy Co-ordinators: mailto:skypy-coordinators@googlegroups.com
+How to contribute
+^^^^^^^^^^^^^^^^^
 
-Contributing
-------------
+Whether you would like to contribute to SkyPy with your own piece of code or
+helping SkyPy develop a concrete feature of the SkyPy roadmap:
 
-We love contributions! SkyPy is open source,
-built on open source, and we'd love to have you hang out in our community.
-For information on how to contribute see our `Contributor Guidelines`_.
+1. Read through our `Discussions Page`_ to start a new conversation and share your
+ideas or follow up an existing conversation on a feature of the roadmap.
+
+2. Upon agreement, open an issue describing the feature you will contribute with and 
+follow the `Contributor Guidelines`_.
+
```

Comment:
Change this section to more closely match the suggestion for the contributor guidelines. e.g.

```suggestion
1. Read through our `Discussions Page`_ to start a new conversation and share your
ideas or follow up an existing conversation on a feature of SkyPy.

2. Following the discussions, when you have a good idea of the specifics of the feature you wish to contribute, open an `Issue`_ describing the feature. Then follow the rest of the `Contributor Guidelines`_ to open a Pull Request to contribute the code implementing the new feature.
```